deploy/crds/rhjmc.redhat.com_containerjfrs_crd.yaml

```diff
@@ -68,6 +66,142 @@ spec:
             properties:
               minimal:
                 type: boolean
+              storageOptions:
+                description: Options to customize the storage for Flight Recordings
+                  and Templates
+                properties:
+                  pvcSpec:
+                    description: Spec for a Persistent Volume Claim, whose options
+                      will override the defaults used by the operator. Unless overriden,
+                      the PVC will be created with the default Storage Class and 500MiB
+                      of storage. Once the operator has created the PVC, changes to
+                      this field have no effect.
```

Comment:
This is almost certainly something for a future enhancement, but I think it would be best if changing this field resulted in the Operator migrating the data from the old PVC to a new one matching the new spec, and then removing the old PVC. Otherwise, end users will end up having to do something like download every recording and template locally, undeploy the operator, redeploy it with a patched pvcSpec, and then re-upload everything.

Reply:
That sounds like a useful feature. I filed #167.